src/app.ts

```diff
@@ -27,6 +27,7 @@ app.use(helmetInstance);
 app.use(corsHandler);
 app.use(requestLogger);
 app.use(expressEJSLayouts);
+app.use(express.json());
```

Comment:
@GarfDev Why was this one necessary here? The same `json()` middleware is currently provided for the account router on line 35. 

I guess this will also cause `oidc-provider` to log warnings since `oidc.callback` also comes with its own json parser. I expect a similar issues as this one with this change. https://github.com/panva/node-oidc-provider/issues/965